pillar.example

```diff
@@ -1,4 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
 systemd:
+  ## tofs pattern
+  tofs:
+    files:
+      networkd:
+        - 'networkd'
+      resolved:
+        - 'resovled.conf'
+      timesyncd:
+        - 'timesyncd.conf'
+
```

Comment:
I suggest the following content here:

```yaml
  tofs:
    # The files_switch key serves as a selector for alternative
    # directories under the formula files directory. See TOFS pattern
    # doc for more info.
    # Note: Any value not evaluated by `config.get` will be used literally.
    # This can be used to set custom paths, as many levels deep as required.
    files_switch:
      - any/path/can/be/used/here
      - id
      - osfinger
      - os
      - os_familly
    # All aspects of path/file resolution are customisable using the options below.
    # This is unnecessary in most cases; there are sensible defaults.
    # dirs:
    #   files: files_alt
    #   default: default_alt
    # files:
    #   networkd:
    #     - 'network'
    #   resolved:
    #     - 'resolved.conf'
    #   timesyncd:
    #     - 'timesyncd.conf'
```

We don't really want anyone setting anything except for `files_switch`.  The defaults are best otherwise.

Reply:
I like it will update the pr.

Reply:
Thanks.